frontend/src/metabase/components/form/FormField/FormFieldView.tsx

```diff
@@ -78,6 +90,14 @@ function FormFieldView({
         </FieldContainer>
       )}
       {align !== "left" && <InputContainer>{children}</InputContainer>}
+      {name === "password" && typeof value === "string" && isRegister && (
```

Comment:
`FormFieldView` is a generic component that decorates form fields with common elements such as field label or field description. It would not be great to extend it with an element that is specific only to password fields.

A better approach would be to add a new FormWidget type, check out `frontend/src/metabase/components/form/FormWidget.jsx`

Reply:
@alxnddr this has been updated now, please review again